src/hub/app.py

```diff
@@ -116,6 +141,8 @@ def display_subhub_errors(e: SubHubError):
 
     @app.app.before_request
     def before_request():
+        logger.debug("Request headers", headers=request.headers)
```

Comment:
This will at the `DEBUG` log level, dump out the AWS `RequestId` along with XRay `Correlation-Id`.  We can explicitly add in specific fields as a `INFO` log level, etc.  Your pick.

Reply:
DEBUG seems appropriate to me

Reply:
Cool!